dev/tools/grunt/tasks/static.js

```diff
@@ -17,7 +17,7 @@ module.exports = function (grunt) {
         };
 
     grunt.registerTask('static', function (target) {
-        var currentTarget = target || 'test',
+        var currentTarget = target || 'file',
```

Comment:
UX. Dev should be able to quickly run `grunt static` to get some instant, readable feedback. Right now will get some weird XML output in middle of nowhere (`dev/tests/static/jscs-error-report.xml`).